README.md

```diff
@@ -1,8 +1,47 @@
+
+**Competent:** Python, Flutter/Dart, Objective C, Firebase, Assembly Language
+
+**Familiar:** HTML/CSS, MATLAB, JavaScript, Android Studio, Node.js
+
+## Jarvis Projects
+
+Project source code: [https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai)
+
+
+**Linux Cluster Resource Monitoring App** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/linux_sql)]: Developed a monitoring agent using Bash scripts that keep an active track of resource usage statistics of a node in a network. Postgres container mounted on a Docker engine is used as a Database. SQL queries were used for data analysis which also includes an error detection query to check if crontab is updating values on a timely basis. The Bash Scripts are automated using crontab. The agent was run in a CentOS 7 virtual machine on the Google Cloud Platform.
+
+**Core Java Apps** [[GitHub](https://github.com/jarviscanada/jarvis_data_eng_HarikrishnaSPillai/tree/master/core_java)]:
+      
+  - JDBC App: Worked on a Java application using JDBC API to connect to a PostgreSQL database. The app allows users to perform CRUD operations and is built using the DAO(Data Access Object) pattern. Maven is used as Project Management Tool.
+  - Grep App: Reimplemented the popular LINUX grep command as a Java App. The application recursively searches through a given directory for a supplied Regex pattern. The matched lines are then written in a text file. The application was improvised by Java 8 features Lambda and Stream API. Streams and Lambda helped in reducing memory usage and inducing a declarative programming approach. IntelliJ was used as IDE and the project was built using Maven. The app is deployed on DockerHub using Docker.
+
+
+## Highlighted Projects
+**Real-Time Parking Detection System**: Designed and Developed a Parking Detection system in a Raspberry Pi and Intel Neural Compute Stick. The system used Yolo v3 Real-time Object detection algorithm to process images supplied by Pi cam. The number of parking spots was conveyed from Raspberry Pi to Firebase using Python and Web API. The data received in real-time was broadcast to the client-side web application. The web app was developed using Flutter and Dart programming language. Additionally, the web app used google authentication API and Map API for authentication and GUI purposes.
+
+**Portfolio Website**: Developed a Responsive website using tech HTML/CSS/JS as front end stack. The back-end was configured using AWS services like S3 and route 53. www.analysthari.com
+
+
+## Professional Experiences
+
+**Software Developer, Jarvis (2020-present)**: Practised Scrum framework for team collaboration and SDLC. Developed several application using technologies such as Docker, Bash scripting, Java, RDBMS(SQL), Project management using Maven and version control using Git/GitHub.
```

Comment:
https://github.com/jarviscanada/jarvis_data_eng_NarimanAlimuradov
https://github.com/jarviscanada/jarvis_data_eng_JasmeetBrar
https://github.com/jarviscanada/jarvis_data_eng_AlexKirilenko
https://github.com/jarviscanada/jarvis_data_eng_FatimaAnwer
https://github.com/jarviscanada/jarvis_data_eng_BrandonDicicco
https://github.com/jarviscanada/jarvis_data_eng_JohnCao
https://github.com/jarviscanada/jarvis_data_eng_TomasRotbauer
https://github.com/jarviscanada/jarvis_data_eng_TinaLee

Reply:
Increased the word count in Experiences